tests/e2e/test_e2e_template.py

```diff
@@ -255,11 +255,11 @@ def _run_import_code_in_notebooks_test() -> None:
     )
     job_id = parse_job_id(out)
 
-    expected_string = "----\r\nHello World!\r\n----"
+    expected_string = "----\r\nYour training script here\r\n----"
 
     out_file = f"/tmp/out-nbconvert-{MK_PROJECT_SLUG}"
     jupyter_nbconvert_cmd = "jupyter nbconvert --execute --no-prompt --no-input"
-    notebook_path = f"{MK_PROJECT_PATH_ENV}/{MK_NOTEBOOKS_DIR}/Untitled.ipynb"
+    notebook_path = f"{MK_PROJECT_PATH_ENV}/{MK_NOTEBOOKS_DIR}/hello_world.ipynb"
```

Comment:
if we have `train.py` in the code dir, let's also have a reasonable name in notebook?

Reply:
I'm all for a reasonable name. However, I think that `hello_world` is not really reasonable here. Let's rename it to `demo`.

The logic is the following: if you have `train.py`, you can simply add code there having a file with a meaningful name. If you have `hello_world.ipynb`, you will have to rename it in a real project, while `demo.ipynb` is a reasonable name for the notebook with model inference.

Reply:
Totally agree, thank you!